content/quickstart.md

```diff
@@ -17,21 +17,96 @@ toc: true
 weight: 20
 ---
 
-<center>
-<iframe width="560" height="315" src="https://www.youtube.com/embed/9vaVMbsHJqk?rel=0" frameborder="0" allowfullscreen></iframe>
-</center>
+You can try out Syndesis very easily locally, too.
+All you need is a [Minishift](https://www.openshift.org/minishift/) installation which is available for all the  major operating systems (Linux, OS X and Windows).
+The following examples assume that you have Minishift installed and can be called with `minishift` from the command line (i.e. `minishift` is to be supposed to be findable in your search path)
 
-We aim to make it as simple as possible for users to try out Syndesis. If you want to try out locally on your laptop, follow the [Using Minishift]({{< relref "#using-minishift" >}}) instructions.
 
-- - -
+## Rocket launch
 
-## Using Minishift
+To make the installation super easy, on OS X and Linux you can use the following command for a complete installation of Syndesis on Minishift in a namespace `syndesis`:
 
-Minishift is a tool that helps you run OpenShift locally by running a single-node OpenShift cluster inside a VM locally. With Minishift you can easily try out Syndesis without requiring a "real" cluster or servers, a laptop will suffice.
+```bash
+bash <(curl -sL https://syndes.is/start)
+```
+
+This will take some time until all images are downloaded and installed but eventually your browser should open with the landing page for Syndesis.
+
+```
+=====================================================================
+SYNDESIS 1.3 QUICKSTART
+
+Hybrid integration on OpenShift made easy
+=====================================================================
+
+---------------------------------------------------------------------
+Starting minishift ....
+
+-- Starting profile 'minishift'
+-- Checking if requested OpenShift version 'v3.7.1' is valid ... OK
+-- Checking if requested OpenShift version 'v3.7.1' is supported ... OK
+-- Checking if requested hypervisor 'xhyve' is supported on this platform ... OK
+-- Checking if xhyve driver is installed ...
+   Driver is available at /usr/local/bin/docker-machine-driver-xhyve
+   Checking for setuid bit ... OK
+-- Checking the ISO URL ... OK
+....
+....
+Sleeping 10s ...
+syndesis-ui-1-kz5qx   0/1       Running   0         34s
+syndesis-ui-1-kz5qx   1/1       Running   0         40s
+syndesis-ui-1-deploy   0/1       Completed   0         46s
+---------------------------------------------------------------------
+Opening http://syndesis.192.168.64.56.nip.io
+```
+
+Here you will be first asked twice to add a Security exception for Minishift's self signed certificate. Please allow this exception:
```

Comment:
Is there a reason we are capitalizing Security? Just curious!

Reply:
Also should be "self-signed"

Reply:
no particular reason, maybe the only excuse is that we capitalized Everything Here In Germany ;-) (and maybe also a reference to the screenshot). But right, should be lowercase